### PR TITLE
Monitoring: Navigation

### PR DESCRIPTION
**Pull request by neotyk**

### CODE REVIEW

File: src/cljs/athens/listeners.cljs [hunk 1]
```diff
@@ -115,15 +115,31 @@
...
-                                          KeyCodes.T     (dispatch [:theme/toggle])
-                                          nil))))
+      alt                                (condp = key-code
+                                           KeyCodes.D (do
+                                                        (rf/dispatch [:reporting/navigation {:source :kbd-alt-d
+                                                                                             :target :home
+                                                                                             :pane   :main-pane}])
+                                                        (router/nav-daily-notes))
+                                           KeyCodes.G (do
+                                                        (rf/dispatch [:reporting/navigation {:source :kbd-ctrl-g
```
Comment (tangjeff0) [inline src/cljs/athens/listeners.cljs:132]: `:kbd-alt-g`

File: src/cljs/athens/views/blocks/textarea_keydown.cljs [hunk 2]
```diff
@@ -598,7 +598,13 @@
                                        (nil? (re-find #"(?s)\]" link)))
                                   (let [eid (db/e-by-av :node/title link)]
                                     (if eid
-                                      (router/navigate-page link e)
+                                      (do
+                                        (rf/dispatch [:reporting/navigation {:source :kbd-ctrl-open
```
Comment (tangjeff0) [inline src/cljs/athens/views/blocks/textarea_keydown.cljs:602]: `:kbd-ctrl-o`

File: src/cljs/athens/views/blocks/textarea_keydown.cljs [hunk 3]
```diff
@@ -622,9 +634,21 @@
                                        (nil? (re-find #"(?s)\(" block-ref))
                                        (nil? (re-find #"(?s)\)" block-ref))
                                        (db/e-by-av :block/uid block-ref))
-                                  (router/navigate-uid block-ref e)
-
-                                  :else (router/navigate-uid uid e))))))
+                                  (do
+                                    (rf/dispatch [:reporting/navigation {:source :kbd-ctrl-open
```
Comment (tangjeff0) [inline src/cljs/athens/views/blocks/textarea_keydown.cljs:638]: `:kbd-ctrl-o`

File: src/cljs/athens/views/blocks/textarea_keydown.cljs [hunk 4]
```diff
@@ -622,9 +634,21 @@
...
+                                                                         :target :block
+                                                                         :pane   (if shift
+                                                                                   :right-pane
+                                                                                   :main-pane)}])
+                                    (router/navigate-uid block-ref e))
+
+                                  :else (do
+                                          (rf/dispatch [:reporting/navigation {:source :kbd-ctrl-open
```
Comment (tangjeff0) [inline src/cljs/athens/views/blocks/textarea_keydown.cljs:646]: `:kbd-ctrl-o`

File: src/cljs/athens/views/left_sidebar.cljs [hunk 5]
```diff
@@ -141,35 +147,40 @@
...
-           [shortcut-component sh]))]
-
-      ;; LOGO + BOTTOM BUTTONS
-      [:footer (use-sub-style left-sidebar-style :footer)
-       [:a (use-style notional-logotype-style {:href "https://github.com/athensresearch/athens/issues/new/choose" :target "_blank"}) "Athens"]
-       [:h5 (use-style {:align-self "center"})
-        [:a (use-style version-style {:href "https://github.com/athensresearch/athens/blob/master/CHANGELOG.md"
-                                      :target "_blank"})
-         (athens.util/athens-version)]]]]]))
+    (fn []
```
Comment (tangjeff0) [inline src/cljs/athens/views/left_sidebar.cljs:160]: why wrap all the hiccup in a `fn`?

**Review by tangjeff0 — COMMENTED** [review]
Do you think it'd be helpful to have a helper function so we don't write 
```
(if shift?
  :right-pane
  :main-pane)
```

eveyrwhere?

File: src/cljs/athens/views/left_sidebar.cljs (hunk 5 above)
Comment (neotyk) [reply src/cljs/athens/views/left_sidebar.cljs:160]: so it's a clear distinction between setup and rendering fn, returning fn from component is making this fn a rendering fn.

File: src/cljs/athens/views/pages/node_page.cljs [hunk 6]
```diff
@@ -431,15 +437,23 @@
             (for [[group-title group] linked-refs]
               [:div (use-style references-group-style {:key (str "group-" group-title)})
                [:h4 (use-style references-group-title-style)
-                [:a {:on-click #(router/navigate-page (parse-renderer/parse-title group-title) %)}
+                [:a {:on-click (fn [e]
+                                 (let [shift?       (.-shiftKey e)
+                                       parsed-title (parse-renderer/parse-title group-title)]
+                                   (rf/dispatch [:reporting/navigation {:source :main-page-linked-refs
```
Comment (tangjeff0) [inline src/cljs/athens/views/pages/node_page.cljs:443]: isn't it possible for this to come from `:right-pane`?

File: src/cljs/athens/views/pages/node_page.cljs [hunk 7]
```diff
@@ -482,7 +496,15 @@
             (for [[[group-title] group] @unlinked-refs]
               [:div (use-style references-group-style {:key (str "group-" group-title)})
                [:h4 (use-style references-group-title-style)
-                [:a {:on-click #(router/navigate-page (parse-renderer/parse-title group-title) %)}
+                [:a {:on-click (fn [e]
+                                 (let [shift?       (.-shiftKey e)
+                                       parsed-title (parse-renderer/parse-title group-title)]
+                                   (rf/dispatch [:reporting/navigation {:source :main-unlinked-refs
```
Comment (tangjeff0) [inline src/cljs/athens/views/pages/node_page.cljs:502]: same as earlier: isn't it possible for this to come from `:right-pane`?

**Review by tangjeff0 — CHANGES_REQUESTED** [review]

**Comment by neotyk** [comment]
> Do you think it'd be helpful to have a helper function so we don't write
> 
> ```
> (if shift?
>   :right-pane
>   :main-pane)
> ```
> 
> eveyrwhere?

Yes, a bit verbose, but will do just fine for now.
Otherwise we'd have to pass along the event itself, which isn't always available.

File: src/cljs/athens/electron/monitoring/core.cljs [hunk 8]
```diff
@@ -33,3 +33,20 @@
...
+                                           :target target
+                                           :pane   pane}}}))
+
+
+(rf/reg-event-fx
+  :reporting/page.create
+  (fn [{:keys [_db]} [_ {:keys [source count]}]]
+    {:posthog/capture-event! {:event-name "feature/page.create"
+                              :opts-map   {:source        source
+                                           :pages-created count}}}))
```
Comment (tangjeff0) [inline src/cljs/athens/electron/monitoring/core.cljs:52]: Want to add :page.create stuff to a separate PR?

**Review by tangjeff0 — APPROVED** [review]

**Pull request merged**